src/pages/OrganizationList.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { withOidcSecure } from '@axa-fr/react-oidc';
+import { PanelGrid, PanelGridItem } from '../components/PanelGrid';
+import { OrganizationListTablePanel } from '../components/OrganizationListTablePanel';
+import {
+	ORGANIZATIONS_DEFAULT_COUNT,
+	ORGANIZATIONS_DEFAULT_PAGE,
+	useOrganizations,
+} from '../pdc-api';
+
+const OrganizationListLoader = () => {
+	const [organizations] = useOrganizations(
+		ORGANIZATIONS_DEFAULT_PAGE,
+		ORGANIZATIONS_DEFAULT_COUNT,
+	);
+
+	useEffect(() => {
+		document.title = 'Organization List - Philanthropy Data Commons';
```

Comment:
I'd call this "Organizations" rather than "Organization List" on user-facing things.